modules/monitoring-persistent-storage-prerequisites.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * observability/monitoring/configuring-the-monitoring-stack.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-prerequisites_{context}"]
+== Persistent storage prerequisites
+
+ifdef::openshift-dedicated,openshift-rosa[]
+* Use the block type of storage.
+endif::openshift-dedicated,openshift-rosa[]
+
+ifndef::openshift-dedicated,openshift-rosa[]
+* Dedicate sufficient persistent storage to ensure that the disk does not become full.
+
+* Use `Filesystem` as the storage type value for the `volumeMode` parameter when you configure the persistent volume.
++
+[IMPORTANT]
+====
+* Do not use a raw block volume, which is described with `volumeMode: Block` in the `PersistentVolumeClaim` object. Prometheus cannot use raw block volumes.
```

Comment:
```suggestion
* Do not use a raw block volume, which is described with `volumeMode: Block` in the `PersistentVolume` resource. Prometheus cannot use raw block volumes.
```
(as for the monitoring components, users don't create `PersistentVolumeClaim` directly)